core/templates/dev/head/components/forms/schema_editors/SchemaBasedBoolEditorDirective.js

```diff
@@ -27,7 +27,7 @@ oppia.directive('schemaBasedBoolEditor', [
         labelForFocusTarget: '&'
       },
       templateUrl: UrlInterpolationService(
-                   '/forms/schema_editors/Schema_based_bool_editor_directive.html'),
+        '/forms/schema_editors/Schema_based_bool_editor_directive.html'),
```

Comment:
done. 

I realized that you have done changes assuming that all directives belong to the /components in your example PR. right ?

So, we can't use getComponentUrl for anylthing outside /components and have to create something like getPagesUrl for directives belonging to /pages/admin. right ?

And app.yaml has to be updated accordingly ?